packages/ia-topnav/src/styles/signed-out-dropdown.js

```diff
@@ -5,27 +5,27 @@ export default css`
     .initial,
     .closed,
     .open {
-      right: 33.7rem;
+      right: 34rem;
     }
 
     .search-hidden.initial,
     .search-hidden.closed,
     .search-hidden.open {
-      right: 18.3rem;
+      right: 18.5rem;
     }
   }
 
   @media (min-width: 990px) {
     .initial,
     .closed,
     .open {
-      right: 39.7rem;
+      right: 40rem;
     }
 
     .search-hidden.initial,
     .search-hidden.closed,
     .search-hidden.open {
-      right: 23.5rem;
+      right: 24.5rem;
```

Comment:
Q: It seems like these adjustments are only a few pixels each, but the error looks like it's offset by 100-200px. How are they shifting that menu over so far?